Commandant.xcodeproj/project.pbxproj

```diff
@@ -414,9 +415,9 @@
 				CLANG_WARN_SUSPICIOUS_MOVE = YES;
 				CURRENT_PROJECT_VERSION = 1;
 				ENABLE_TESTABILITY = YES;
-				MACOSX_DEPLOYMENT_TARGET = 10.9;
+				MACOSX_DEPLOYMENT_TARGET = 10.10;
```

Comment:
Is this change necessary? I know that Nimble requires 10.10, but we should be able to adjust the deployment target for just the test target and leave the framework so it can be deployed to 10.9.